pkg/reconciler/sinkbinding/controller.go

```diff
@@ -181,14 +186,17 @@ func (s *SinkBindingSubResourcesReconciler) Reconcile(ctx context.Context, b psb
 		}, b)
 	}
 	
+	// Add feature flags to context 
+	ctx = s.featureStore.ToContext(ctx)
+
 	featureFlags := feature.FromContext(ctx)
```

Comment:
```suggestion
	featureFlags := s.featureStore.Load()
```
You can use this form. So no need to add it to the context and read it from it